go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/falcosecurity/testing
 
-go 1.17
+go 1.21
```

Comment:
why do we need this?

Reply:
I wrote it in the commit message sorry, basically the docker package needs go 1.21

Reply:
It was working before because the CI had 
```
 - name: Setup Go
      uses: actions/setup-go@v3
      with:
        go-version: >= 1.17
```

and it installed latest go version basically.

Reply:
got it!